passkey-authenticator/src/authenticator/get_assertion.rs

```diff
@@ -107,6 +107,17 @@ where
         //        the authenticator), terminate this procedure and return the
         //        CTAP2_ERR_OPERATION_DENIED error.
 
+        // [WebAuthn-9]. Increment the credential associated signature counter or the global signature
+        //               counter value, depending on which approach is implemented by the authenticator,
+        //               by some positive value. If the authenticator does not implement a signature
+        //               counter, let the signature counter value remain constant at zero.
+        if let Some(counter) = credential.counter {
+            credential.counter = Some(counter + 1);
```

Comment:
I don't think this is a realistic concern, but does the spec say anything about what to do if the counter overflows?

Reply:
Huh, I've never consider this, apparently not, can't find any mention of it 🤔

Reply:
I guess nobody expects me to authenticate 4,294,967,295 times 😁